### DISCUSSION TITLE
Data not refreshing when hitting back button with useAsyncData-style call with transform

### DISCUSSION THREAD

**Discussion by belfortf** [johannschopplich/nuxt-api-party#65] [Q&A]
Hi Johann,

I guess this is not something specific from nuxt-api-party but rather from useAsyncData. Here's my call:

``` javascript
const { data, pending, error, refresh } = await useGraphqlamplifyData("/",
    {
        key: slug.value,
        method: "POST",
        body: JSON.stringify({
            nameGraphqlFunction: "listProductsBySlug",
            variables: {
                slug_hu: slug.value,
            },
        }),
        transform: res => res,
    },
);
```
This works well, also when I go to another page and then back to this page via the browser's back button. However, if I'm using the transform function to tailor the output a little, I get an error when I use the back button to get back to the page:

``` javascript
transform: res => res.data.listProductsBySlug.items[0]
```
Any ideas why because of the transform the data doesn't refresh ?

Thanks

**Comment by johannschopplich** [comment]
No idea why that happens. 🤔 Can you provide a minimum reproducible example and open an issue, please?
